meetings/2024-11-12.md

```diff
@@ -2,10 +2,77 @@
 
 ## 12th November 2024
 
-Present:
+Attendees: Alex Tilley (Publish What You Fund), Angelina Virchenko (UN Food and Agriculture Organization), Bill Anderson, David Megginson (UN OCHA), Elise Egan (USAID), Mark Brough (Emergentally), Joshua Satre (USAID), Siem Vaessen (Zimmerman + IATI Governing Board), Sylvan Ridderinkhof (Zimmerman), Melis Karakus (Danish Refugee Council)
 
-Regrets:
+IATI Secretariat representatives: Steven Flower (Open Data Services), Bee Webb (Open Data Services), Rob Redpath (Open Data Services), Isabel Precious-Birds (Open Data Services), Cody Wallace (UNDP), Amy Guy (Open Data Services)
 
-### Notes
+Apologies: Annelise Parr (UNDP), Ryan Boles (USAID),  Marcel Laqueur (FFO Germany), Johan Zambrano (USAID), Melina Lower (GIZ)
 
-### Actions
\ No newline at end of file
+Non attendees: Wakilongo Mweshwa Robert (AIGD AFRICA)
+
+### Minutes 
+
+#### Recap of objectives and previous minutes
+
+* [Recap](https://docs.google.com/presentation/d/1N6Xu8GZYte61yE5Gcpaty3rrrZXf_ho1mWmLOjIMPpI/edit?usp=sharing) of objectives of the SSWG from Steven.
+* Previous minutes have been shared in [GitHub](https://github.com/IATI/standard-process/tree/main/meetings).
+* Today is the final meeting of this time bound Standard Stewardship Working Group (SSWG).
+
+#### Reflections on work so far
+
+* Concern about short time to next board meeting and MA, and risk of not changing the standard for another year.
+* Suggestion of small initial changes to the standard, to reintroduce the community to the concept of a changing data standard. 
+* We have principles to work to from this group. 
+
+#### Examples from other standards
+
+Steven shared two examples of change control documents from other data standards,  [360 Giving](https://github.com/IATI/standard-process/discussions/10) and [Open Referral](https://github.com/IATI/standard-process/discussions/13).
+
+* Agreement that a similar template would be useful for IATI.
+* Need to ensure that this process includes communicating changes and their impact to data users.
+* Idea - a requirement to "subscribe" to a IATI Standard Updates listserv in order to download IATI files or embed/use secretariat run APIs?
+* A standard way of logging dependencies would be useful - i.e. which tools would be impacted by a change and how
+* Suggestion of a standard update gazette.
+
+#### Impact assessment
+
+Amy presented a draft [impact-based upgrade approach](https://github.com/IATI/standard-process/discussions/12). This approach would be pragmatic, and decouple changes to the standard from the Major/Minor/Patch definitions.
```

Comment:
```suggestion
Amy presented a draft [impact-based upgrade approach](https://github.com/IATI/standard-process/discussions/12). This approach would be pragmatic, based on real-world impact rather than theoretical or hypothetical, and decouple the decision-making about changes to the standard from the Major/Minor/Patch definitions.
```